<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Form item</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_formName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>form name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_formId">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="CMB_formItemStatus">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Form</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_createFormItem">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Form Item</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_desc">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>99</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
